python3Packages.vllm: 0.10.1.1 -> 0.10.2

daniel-fahey opened the pull request:
Notably adds Python 3.13 support (see https://github.com/vllm-project/vllm/pull/13164) among many other things (https://github.com/vllm-project/vllm/releases/tag/v0.10.2).

Supersedes existing PR #442802 from @r-ryantm.

Python 3.13 support is also needed for vLLM to appear in the builds on the new (and fantastic initiative, btw) [nixos-cuda](https://github.com/nixos-cuda) [Hydra](https://hydra.nixos-cuda.org/), configured by @GaetanLepage.

<details>

<summary>For reference I share part of my engineer's log re: the nixos-cuda builder herein:</summary>

The new [hydra.nixos-cuda.org](https://hydra.nixos-cuda.org/) instance is building the [cuda-packages jobset](https://hydra.nixos-cuda.org/jobset/cuda/cuda-packages/evals), which is defined by [`release-cuda.nix`](https://github.com/NixOS/nixpkgs/blob/db8556be0060cd2f57e58010f95b710b0f4021c7/pkgs/top-level/release-cuda.nix#L168). While `python3Packages.vllm` is explicitly listed in that file, it doesn't appear in any evaluation results.

This is because vLLM is being filtered out during the evaluation phase:
1. The nixpkgs revision being evaluated defaults to Python 3.13 (as of writing, [eval 53](https://hydra.nixos-cuda.org/eval/53) uses [`b6f6c61`](https://github.com/NixOS/nixpkgs/commit/b6f6c613838dd776620c34e8f15fe4d8a9cdf9c0), see below)
2. Our Nixpkgs derivation currently has vLLM 0.10.1.1 with [`disabled = pythonAtLeast "3.13";`](https://github.com/NixOS/nixpkgs/blob/a77b87719ff11d3d7ea2439c406361b6b0c4c56a/pkgs/development/python-modules/vllm/default.nix#L273) in its package definition
3. This causes evaluation to fail with: `error: vllm-0.10.1.1 not supported for interpreter python3.13`

To check which nixpkgs revision the new nixos-cuda Hydra is currently using:
```console
[daniel@laptop:~]$ # Get the nixpkgs revision used in the latest evaluation (53; as above)
curl -sH "Accept: application/json" https://hydra.nixos-cuda.org/eval/53 | \
  jq -r '.jobsetevalinputs.nixpkgs.revision'
b6f6c613838dd776620c34e8f15fe4d8a9cdf9c0
```

To reproduce the evaluation failure:
```console
[daniel@laptop:~]$ # This will fail with the Python 3.13 error
nix-instantiate --eval --expr '
  let
    pkgs = import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/b6f6c613838dd776620c34e8f15fe4d8a9cdf9c0.tar.gz") {
      config = {
        allowUnfree = true;
        cudaSupport = true;
        inHydra = true;
        allowAliases = false;
      };
    };
  in
    pkgs.python3Packages.vllm.drvPath
'
error:
       … while evaluating the attribute 'python3Packages.vllm.drvPath'
         at /nix/store/w2l3qlg0bvkhiy34dkgdmhwq68qk1y8f-source/lib/customisation.nix:429:7:
          428|     // {
          429|       drvPath =
             |       ^
          430|         assert condition;

       … while evaluating the attribute 'drvPath'
         at /nix/store/w2l3qlg0bvkhiy34dkgdmhwq68qk1y8f-source/lib/customisation.nix:429:7:
          428|     // {
          429|       drvPath =
             |       ^
          430|         assert condition;

       (stack trace truncated; use '--show-trace' to show the full, detailed trace)

       error: vllm-0.10.1.1 not supported for interpreter python3.13
```

And to verify vLLM is missing from the evaluation:
```console
[daniel@laptop:~]$ # Check which python3Packages are actually in the latest evaluation
curl -sH "Accept: application/json" https://hydra.nixos-cuda.org/eval/53/builds | \
  jq -r 'to_entries[] | .value.job' | grep python3Packages | sort
python3Packages.cupy.x86_64-linux
python3Packages.faiss.x86_64-linux
python3Packages.faster-whisper.x86_64-linux
python3Packages.flashinfer.x86_64-linux
python3Packages.flax.x86_64-linux
python3Packages.gpt-2-simple.x86_64-linux
python3Packages.grad-cam.x86_64-linux
python3Packages.jaxlib.x86_64-linux
python3Packages.jax.x86_64-linux
python3Packages.keras.x86_64-linux
python3Packages.kornia.x86_64-linux
python3Packages.mmcv.x86_64-linux
python3Packages.mxnet.x86_64-linux
python3Packages.numpy.x86_64-linux
python3Packages.onnx.x86_64-linux
python3Packages.openai-whisper.x86_64-linux
python3Packages.opencv4.x86_64-linux
python3Packages.opensfm.x86_64-linux
python3Packages.pycuda.x86_64-linux
python3Packages.pyrealsense2WithCuda.x86_64-linux
python3Packages.pytorch-lightning.x86_64-linux
python3Packages.scikit-image.x86_64-linux
python3Packages.scikit-learn.x86_64-linux
python3Packages.scipy.x86_64-linux
python3Packages.spacy-transformers.x86_64-linux
python3Packages.tensorflow.x86_64-linux
python3Packages.tesserocr.x86_64-linux
python3Packages.tiny-cuda-nn.x86_64-linux
python3Packages.torchaudio.x86_64-linux
python3Packages.torchvision.x86_64-linux
python3Packages.torch.x86_64-linux
python3Packages.transformers.x86_64-linux
python3Packages.triton.x86_64-linux
python3Packages.ttstokenizer.x86_64-linux
python3Packages.vidstab.x86_64-linux
```
</details>

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 447722 --extra-nixpkgs-config '{ allowUnfree = true; cudaSupport = true; }'`
Commit: `fbc1629b8775a9eeb932f1b79d14fa973adb67ec`

---
### `x86_64-linux`
<details>
  <summary>:x: 12 packages failed to build:</summary>
  <ul>
    <li>python312Packages.kserve</li>
    <li>python312Packages.kserve.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>vllm (python312Packages.vllm)</li>
    <li>vllm.dist (python312Packages.vllm.dist)</li>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vllm.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>nixpkgs-manual</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>vllm</summary>
<pre>_distutils/command/build_ext.py&quot;, line 368, in run
    self.build_extensions()
  File &quot;&lt;string&gt;&quot;, line 240, in build_extensions
  File &quot;&lt;string&gt;&quot;, line 218, in configure
  File &quot;/nix/store/zsbkvanzzx4dd5va9ivsx83rs12d4dsv-python3-3.12.11/lib/python3.12/subprocess.py&quot;, line 413, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command &#x27;[&#x27;cmake&#x27;, &#x27;/build/source&#x27;, &#x27;-G&#x27;, &#x27;Ninja&#x27;, &#x27;-DCMAKE_BUILD_TYPE=RelWithDebInfo&#x27;, &#x27;-DVLLM_TARGET_DEVICE=cuda&#x27;, &#x27;-DFETCHCONTENT_SOURCE_DIR_CUTLASS:STRING=/nix/store/n3bkhnw519zjw9g66l37l1yks6cqri0y-source&#x27;, &#x27;-DFLASH_MLA_SRC_DIR:STRING=/nix/store/ab1x9ra2sc9k9r2nxsrhhcm0izncxbsz-flashmla-1.0.0&#x27;, &#x27;-DVLLM_FLASH_ATTN_SRC_DIR:STRING=/nix/store/s9xqh0qzpk19hx5n0i52r2rf2fpxhw9a-vllm-flash-attn-2.7.4.post1&#x27;, &#x27;-DTORCH_CUDA_ARCH_LIST:STRING=7.5;8.0;8.6;8.9;9.0;10.0;12.0&#x27;, &#x27;-DCUTLASS_NVCC_ARCHS_ENABLED:STRING=75;80;86;89;90;100;120&#x27;, &#x27;-DCUDA_TOOLKIT_ROOT_DIR:STRING=/nix/store/vq0njavy3k2lkxv06yq2g5jq93a44r09-cuda-merged-12.8&#x27;, &#x27;-DCAFFE2_USE_CUDNN:STRING=ON&#x27;, &#x27;-DCAFFE2_USE_CUFILE:STRING=ON&#x27;, &#x27;-DCUTLASS_ENABLE_CUBLAS:STRING=ON&#x27;, &#x27;-DCUDA_HOST_COMPILER=/nix/store/fp97arzydz32c5gln0fia194n5mynwkw-gcc-wrapper-14.3.0/bin/c++&#x27;, &#x27;-DCMAKE_CUDA_HOST_COMPILER=/nix/store/fp97arzydz32c5gln0fia194n5mynwkw-gcc-wrapper-14.3.0/bin/c++&#x27;, &#x27;-DVLLM_PYTHON_EXECUTABLE=/nix/store/zsbkvanzzx4dd5va9ivsx83rs12d4dsv-python3-3.12.11/bin/python3.12&#x27;, &#x27;-DVLLM_PYTHON_PATH=/nix/store/zsbkvanzzx4dd5va9ivsx83rs12d4dsv-python3-3.12.11/lib/python3.12/site-packages:/nix/store/9b3qjfy0c99x2c5asn07xg0lagivk9yd-python3.12-wheel-0.46.1/lib/python3.12/site-packages:/nix/store/b0cx3hggf32q5jx0plwy06b750vlnj46-python3.12-packaging-25.0/lib/python3.12/site-packages:/nix/store/shk3wqjsz0dkknnwh2zy34scz04lfiww-python3.12-installer-0.7.0/lib/python3.12/site-packages:/nix/store/0ib19lfpsajp0g1251108372ha8148c0-python3.12-cmake-3.31.7/lib/python3.12/site-packages:/nix/store/9wmkgzq8fxbdw4dl3b3dimfrmhdfqp46-python3.12-jinja2-3.1.6/lib/python3.12/site-packages:/nix/store/kjh9ih2q1ab03jxfyjwxd0bm99ycrvca-python3.12-markupsafe-3.0.2/lib/python3.12/site-packages:/nix/store/jj2xbhgmrzzc6mhj3vwnr7b366cb142i-python3.12-ninja-1.13.1/lib/python3.12/site-packages:/nix/store/59pcizvg9ihq2kblyvpq1b4qxk0v7c48-python3.12-setuptools-80.9.0/lib/python3.12/site-packages:/nix/store/ch2mvgw547nh77rk453m2wizl6y2msji-python3.12-setuptools-scm-9.0.1/lib/python3.12/site-packages:/nix/store/xlrjrzy12gdcxb1gfyqzvsvcjszfisks-python3.12-typing-extensions-4.14.1/lib/python3.12/site-packages:/nix/store/lxkkvs805af81sf05ys2gff97q7l4abh-python3.12-astunparse-1.6.3/lib/python3.12/site-packages:/nix/store/ki371fxbikc2vm99if7si8az6f8qdkwd-python3.12-six-1.17.0/lib/python3.12/site-packages:/nix/store/5ibk7hk1ldqg0rgmj277z2j7g6ddffdd-python3.12-expecttest-0.2.1/lib/python3.12/site-packages:/nix/store/jzzh3zgb71sbm7n8fz7ganmljsm9vz3s-python3.12-filelock-3.18.0/lib/python3.12/site-packages:/nix/store/b3s308csjpv449pkfm5jklgz70gb7ir1-python3.12-fsspec-2025.3.2/lib/python3.12/site-packages:/nix/store/mmzchg3wlrf1lv96igxwpy4wjnvkcqbj-python3.12-hypothesis-6.136.9/lib/python3.12/site-packages:/nix/store/xm0f2xd6hgqwfs3mql6aifagwpx965ck-python3.12-attrs-25.3.0/lib/python3.12/site-packages:/nix/store/x7zdjr129i5hxzf53a3kknkmzjpfm38l-python3.12-sortedcontainers-2.4.0/lib/python3.12/site-packages:/nix/store/4b2q14br2ly9zbc05bb9yifqadgq4l2y-python3.12-networkx-3.5/lib/python3.12/site-packages:/nix/store/y3pqas1lc3m57k34nadg4fdz14baw4cj-python3.12-psutil-7.0.0/lib/python3.12/site-packages:/nix/store/q40xs60jc85b1ij40784bcf0dqm38l52-python3.12-pyyaml-6.0.2/lib/python3.12/site-packages:/nix/store/zja9psd8n4jkykiyhnmcmd3x2kwsmby4-python3.12-requests-2.32.4/lib/python3.12/site-packages:/nix/store/za74065ycbi3w1rsfg9lk6w4spd8ykwk-python3.12-certifi-2025.07.14/lib/python3.12/site-packages:/nix/store/zs7cga78nj9ndgcy5lajd0rplyaxxqbd-python3.12-charset-normalizer-3.4.3/lib/python3.12/site-packages:/nix/store/29l9yxqip0cin7z0syv5kapwf5zk6j6v-python3.12-idna-3.10/lib/python3.12/site-packages:/nix/store/kjhw20ac0x7z7pkpq2kirkqzla50imp7-python3.12-urllib3-2.5.0/lib/python3.12/site-packages:/nix/store/jy6n9743awvxlncl29dp306s3z3s8c56-python3.12-sympy-1.14.0/lib/python3.12/site-packages:/nix/store/fb0p52dswdnjda8n0im2j6bl1ysg075h-python3.12-mpmath-1.3.0/lib/python3.12/site-packages:/nix/store/ff4cnplr4w742q1rapx2clyv9spvkdlg-python3.12-types-dataclasses-0.6.6/lib/python3.12/site-packages:/nix/store/rzzjid7dh9p15qziqbb4mj7wwsv93r60-python3.12-pillow-11.3.0/lib/python3.12/site-packages:/nix/store/dzanqkrcpfbfm8bl4s5g4mk5hj60n2w1-python3.12-tensorboard-2.20.0/lib/python3.12/site-packages:/nix/store/0gj6dy5shjakgjk8273wirimxdhpwf0a-python3.12-absl-py-2.3.1/lib/python3.12/site-packages:/nix/store/h06x4gwzmgvfxb7l1lyfhgs33cwid9p7-python3.12-protobuf-6.32.0/lib/python3.12/site-packages:/nix/store/ab1diw1siyzlg012bpz559zf6x8lj6xp-python3.12-grpcio-1.74.0/lib/python3.12/site-packages:/nix/store/1r8rz2db82dw5r8b17ilbr1bf5wg6g5h-python3.12-markdown-3.8.2/lib/python3.12/site-packages:/nix/store/c82j3fpgc1545rgl3sbibcwn2964fbmv-python3.12-numpy-2.3.2/lib/python3.12/site-packages:/nix/store/lvd7san898v5z0yv91711rhpgj5ysnlb-python3.12-tensorboard-data-server-0.7.2/lib/python3.12/site-packages:/nix/store/ckj50ws3bm6k8siny3ganwp7inhn25bz-python3.12-werkzeug-3.1.3/lib/python3.12/site-packages:/nix/store/6pa72yhmxs4wq0p0v9abq8k0agsjyr6f-python3.12-pybind11-2.13.6/lib/python3.12/site-packages:/nix/store/vr4a33n5hhlhrglaqxlknsrzpfmg9dn7-python3.12-triton-3.4.0/lib/python3.12/site-packages:/nix/store/lq533scw7p19wr4rs8av6jd8ckah8zm7-python3.12-torch-2.8.0/lib/python3.12/site-packages:/nix/store/dzl9zy8ylzr3ac9krvvicj18q9x1w6jc-python3.12-cffi-1.17.1/lib/python3.12/site-packages:/nix/store/12fiwq778f29ssxp44qa08lx5yd513ws-python3.12-pycparser-2.22/lib/python3.12/site-packages:/nix/store/35kgpvz7fksqd5lqm3z1dprck7w66mfv-python3.12-aioprometheus-unstable-2023-03-14/lib/python3.12/site-packages:/nix/store/hqnqcy6s2fvi0k9yyy6d9r89ha6065qs-python3.12-orjson-3.10.18/lib/python3.12/site-packages:/nix/store/6skddb8bdr1rx28rjfib6mkhb4l6np47-python3.12-quantile-python-1.1/lib/python3.12/site-packages:/nix/store/s2hjjlywpnp3isn0ghv7q1lvyzlnglid-python3.12-blake3-1.0.7/lib/python3.12/site-packages:/nix/store/y37m52q141p3snjdy3kpw1jyscbq3kxa-python3.12-cachetools-6.1.0/lib/python3.12/site-packages:/nix/store/j7rfcbv6di1lwssvrrzkq4dpbhagwxxx-python3.12-cbor2-5.6.5/lib/python3.12/site-packages:/nix/store/v350vkqxc60p8hcadqyjg1kdcghi15jd-python3.12-depyf-0.19.0/lib/python3.12/site-packages:/nix/store/gm3sw5lf73vlm16pc8c3kh00lqnpfvrj-python3.12-astor-0.8.1/lib/python3.12/site-packages:/nix/store/6w0kdidc8d9lmk2rcc12zhwhl7jlzdwc-python3.12-dill-0.4.0/lib/python3.12/site-packages:/nix/store/1gnm78ghn7waqfncmgry6mc38ywqxr3s-python3.12-fastapi-0.116.1/lib/python3.12/site-packages:/nix/store/13b1g9pw8l3nq973sydqm2b590f83iv1-python3.12-starlette-0.47.2/lib/python3.12/site-packages:/nix/store/slh585m8dw0pffhjb36glj1q4l6cy1qm-python3.12-anyio-4.10.0/lib/python3.12/site-packages:/nix/store/hnwlqbmy64hjppgab5i11w49j9aw0jlr-python3.12-sniffio-1.3.1/lib/python3.12/site-packages:/nix/store/s46jim78c8iw97a5yvq9c2z7qvmdw67v-python3.12-pydantic-2.11.7/lib/python3.12/site-packages:/nix/store/pgq7q75fkjdb6s70cb3ivr0n7lx1znlk-python3.12-annotated-types-0.7.0/lib/python3.12/site-packages:/nix/store/n9b7krs42gwq7lv2aa3mz765kr0avkd5-python3.12-pydantic-core-2.33.2/lib/python3.12/site-packages:/nix/store/m548h8kylkm1v3jjpkv0dff3z1k1d18m-python3.12-typing-inspection-0.4.1/lib/python3.12/site-packages:/nix/store/a7qb5cg4i46i9jzl2835lcqlcx5dn60y-python3.12-llguidance-1.2.0/lib/python3.12/site-packages:/nix/store/k5d4bnma12zx3ar65qs5fjipkxqpa167-python3.12-lm-format-enforcer-0.11.3/lib/python3.12/site-packages:/nix/store/bbk5aiyb4ji705h6gcbmvzi7gkdm5zif-python3.12-interegular-0.3.3/lib/python3.12/site-packages:/nix/store/kcjpp59xx644igwhxmg7ihppg8xhglpq-python3.12-openai-1.101.0/lib/python3.12/site-packages:/nix/store/akq7zhqjw6jxzirvickc3canwhxks9xy-python3.12-distro-1.9.0/lib/python3.12/site-packages:/nix/store/dm0hjj6q6g3m2qqv4askl15i36yd4w47-python3.12-httpx-0.28.1/lib/python3.12/site-packages:/nix/store/dgiww3l8m52pj1cnzjjj59sxnr5k0294-python3.12-httpcore-1.0.9/lib/python3.12/site-packages:/nix/store/h66byw0853jyr01z99ln367l7zvp6vg0-python3.12-h11-0.16.0/lib/python3.12/site-packages:/nix/store/avl0pkrc16yair3j7d0dlrj6p79bj71v-python3.12-jiter-0.8.2/lib/python3.12/site-packages:/nix/store/y76h0grv88jlvcy28kf7j1qwpgf7fygm-python3.12-tqdm-4.67.1/lib/python3.12/site-packages:/nix/store/mcjbdf6nzhgjyv71r20lqh0793vysqsd-python3.12-aiohttp-3.12.15/lib/python3.12/site-packages:/nix/store/88gyn1gms5yyd3l5wafqgd6n7dqkh3y4-python3.12-aiohappyeyeballs-2.6.1/lib/python3.12/site-packages:/nix/store/4mvb9kz2xszn1aipb53djx0s96pl2kb4-python3.12-aiosignal-1.4.0/lib/python3.12/site-packages:/nix/store/3c6lczyzx7gfy3ma53zm0iw76vzyr2bg-python3.12-frozenlist-1.7.0/lib/python3.12/site-packages:/nix/store/6wrh26sbnx3xcylm8zghf6abw0jszwzi-python3.12-async-timeout-5.0.1/lib/python3.12/site-packages:/nix/store/8gi5nlvfnf55s2a1y63nzaxfymdq9pw9-python3.12-multidict-6.6.4/lib/python3.12/site-packages:/nix/store/jrb5lhkygz0awxl2s0nwaa6d8p7749s5-python3.12-propcache-0.3.2/lib/python3.12/site-packages:/nix/store/sq6bymah55dg3vvybf88sx0zmcki6paj-python3.12-yarl-1.21.1/lib/python3.12/site-packages:/nix/store/psqmbxqasz7x5w7c01c6jn1iibb319kf-python3.12-aiodns-3.5.0/lib/python3.12/site-packages:/nix/store/w0561jiy0mh692vqib9zj1l21iiml47n-python3.12-pycares-4.9.0/lib/python3.12/site-packages:/nix/store/pkfya6f5rf80cpdxwj9n5fzqlgign1dg-python3.12-brotli-1.1.0/lib/python3.12/site-packages:/nix/store/hrznn59qlch53id761y7xdb3i9lahw5l-python3.12-httpx-aiohttp-0.1.8/lib/python3.12/site-packages:/nix/store/hp0g650v0b0zjxxf462rmsi1qfv51jpg-python3.12-websockets-15.0.1/lib/python3.12/site-packages:/nix/store/irpdll8f799mqzg76ra9a0ywzah7ys6g-python3.12-sounddevice-0.5.2/lib/python3.12/site-packages:/nix/store/qvhzmx3q06asbf1b57f787sx0k4r9cm3-python3.12-opencv-python-headless-4.12.0/lib/python3.12/site-packages:/nix/store/w390sqm25kqdwx387n3abp9m934259s8-opencv-4.12.0/lib/python3.12/site-packages:/nix/store/ykypgrz197xnl5mgrzq17h8kg60djpwj-python3.12-outlines-1.2.3/lib/python3.12/site-packages:/nix/store/gb7vr44n2za477k90rr4kjappp82x5pj-python3.12-airportsdata-20250909.5/lib/python3.12/site-packages:/nix/store/f2w6qmr6jgndc7sgv2289a5l9d1mfzs6-python3.12-cloudpickle-3.1.1/lib/python3.12/site-packages:/nix/store/nmyfmmi15rwzr59hc9y636jwngjm3p6w-python3.12-datasets-4.0.0/lib/python3.12/site-packages:/nix/store/ahq4cyl9aq9wnxd01a0m9ibki3w0k2cn-python3.12-huggingface-hub-0.35.0/lib/python3.12/site-packages:/nix/store/kdv4acw6i7w3qm827gdcq1hlzg8xhzlb-python3.12-hf-xet-1.1.10/lib/python3.12/site-packages:/nix/store/9ixhzj3jrk5g91sa020y3c8mg61skbsk-python3.12-multiprocess-0.70.18/lib/python3.12/site-packages:/nix/store/av6gkli762sw8kp3z6p8yyrspd608ymq-python3.12-pandas-2.3.1/lib/python3.12/site-packages:/nix/store/3ld79kvyhqn6ngyc1d3ykpbm8jy03ffp-python3.12-python-dateutil-2.9.0.post0/lib/python3.12/site-packages:/nix/store/6wvi6gmi686pgx74fqnac94d24xlx4ji-python3.12-pytz-2025.2/lib/python3.12/site-packages:/nix/store/awx3n5809rq29yiq8nlpd7a7a0ha2h9p-python3.12-tzdata-2025.2/lib/python3.12/site-packages:/nix/store/v3lgpjfgbwa9wm8yjfnb12iqgl35mgn6-python3.12-pyarrow-20.0.0/lib/python3.12/site-packages:/nix/store/y8j01xsa6s6hhscxyfg2igdys8s82phw-python3.12-responses-0.25.7/lib/python3.12/site-packages:/nix/store/n7armn49h3fwvmlqig83v9vi7nxis5ff-python3.12-types-pyyaml-6.0.12.20250516/lib/python3.12/site-packages:/nix/store/kcwkadsvd61mvsrkyciklr67jz5d2vcp-python3.12-types-toml-0.10.8.20240310/lib/python3.12/site-packages:/nix/store/v7k2qz5lhq1ihs4cynhwg481mhgg25c5-python3.12-xxhash-3.5.0/lib/python3.12/site-packages:/nix/store/9367y1v8505dijsffq8zd9v1s70n5nm0-python3.12-diskcache-5.6.3/lib/python3.12/site-packages:/nix/store/2gy7pvazrbkqfilkdjb7vkxw1d3mcap1-python3.12-genson-1.3.0/lib/python3.12/site-packages:/nix/store/2x614n745xklwmd196sdbsjhd6zxdm7w-python3.12-iso3166-2.1.1/lib/python3.12/site-packages:/nix/store/nmb2myz57hz6wl99m24amr0155kyd7b6-python3.12-jsonschema-4.25.0/lib/python3.12/site-packages:/nix/store/623lfsnqzm4ywq1pyxhighbcnfpcq0w4-python3.12-jsonpath-ng-1.7.0/lib/python3.12/site-packages:/nix/store/fsdylkpndwrsnphal23p2kkxpcm6g4gk-python3.12-ply-3.11/lib/python3.12/site-packages:/nix/store/n6g5m6f70fzkrbjxhbg9cv4y93bhm7ys-python3.12-jsonschema-specifications-2025.4.1/lib/python3.12/site-packages:/nix/store/dlipy4q4b807yplqh7hzmwf8m5ajxp9v-python3.12-referencing-0.36.2/lib/python3.12/site-packages:/nix/store/y1cx16nyhzgkpzsa2aqf9rdvv2lc8bcb-python3.12-rpds-py-0.25.0/lib/python3.12/site-packages:/nix/store/8yrdady4yxbg96v4grp3bfdlf18ylgh0-python3.12-lark-1.2.2/lib/python3.12/site-packages:/nix/store/i4ab965mnnvf2ls4ygdgjrgc7nl4shxl-python3.12-regex-2025.7.34/lib/python3.12/site-packages:/nix/store/503jip8bjlq3xija5bppk826qzsxjzxl-python3.12-nest-asyncio-1.6.0/lib/python3.12/site-packages:/nix/store/y6190ffbavm26sjcmgk23w5h9kink5i4-python3.12-outlines-core-0.2.11/lib/python3.12/site-packages:/nix/store/nqqcindzdv7pnvxqdg324j1ks33i2kyb-python3.12-pycountry-24.6.1/lib/python3.12/site-packages:/nix/store/iy40dqxha3bdsvxxxlcgckrmq864rhqh-python3.12-transformers-4.56.2/lib/python3.12/site-packages:/nix/store/qngkhbgyw8cm1yx7a4xwgxwj98dk2acv-python3.12-tokenizers-0.22.1/lib/python3.12/site-packages:/nix/store/sk3a78xrqw8pl1l6jrlxi44apxmjld2i-python3.12-safetensors-0.6.2/lib/python3.12/site-packages:/nix/store/kp0jpcnlnapqaigdz9sid3019ix6q712-python3.12-prometheus-fastapi-instrumentator-7.1.0/lib/python3.12/site-packages:/nix/store/yldc5sisslcgy8w4nsswk6p78pkx9iqw-python3.12-prometheus-client-0.22.1/lib/python3.12/site-packages:/nix/store/glfil8nfv9217pv7avjx92myjcm0idd5-python3.12-py-cpuinfo-9.0.0/lib/python3.12/site-packages:/nix/store/yk3r7accpdnggyfj1dsrgj50p5ckg6m9-python3.12-pybase64-1.4.2/lib/python3.12/site-packages:/nix/store/fsb3swvaz80njmhklpif6j0wpnjm64ra-python3.12-python-json-logger-3.3.0/lib/python3.12/site-packages:/nix/store/3d7fg1f2fn546li4cnpm4n08j9pxvphc-python3.12-python-multipart-0.0.20/lib/python3.12/site-packages:/nix/store/6wc4ghak1j9l2vg76m3gglq3d6gh1fag-python3.12-pyzmq-27.0.1/lib/python3.12/site-packages:/nix/store/l6yxmy4k4kgpri4mzv6rgpacmxv3j89n-python3.12-ray-2.49.2/lib/python3.12/site-packages:/nix/store/9m1rr6cipdbi89v4yqfmbbqhl1b4y87h-python3.12-click-8.1.8/lib/python3.12/site-packages:/nix/store/dc6khwkvk6rx6dn1v03y6k802y3s37jk-python3.12-msgpack-1.1.1/lib/python3.12/site-packages:/nix/store/532j0dmsvnarks12dgbnzhy62pgnnc4b-python3.12-watchfiles-1.0.5/lib/python3.12/site-packages:/nix/store/vziamkjx20yi7b01k489zarz2zb9nz6i-python3.12-sentencepiece-0.2.1/lib/python3.12/site-packages:/nix/store/42l37a4vhwc7gdyqzfy934shhd0r3sr0-python3.12-tiktoken-0.9.0/lib/python3.12/site-packages:/nix/store/75pl5hln81m5hd6myb3miklfwjyfhbpx-python3.12-blobfile-3.0.0/lib/python3.12/site-packages:/nix/store/sxwld68cw57ybmq6rw42nb5w8a8klzzc-python3.12-pycryptodomex-3.23.0/lib/python3.12/site-packages:/nix/store/ipqbahg79q5mz5w5glq9b0i4k08xs079-python3.12-lxml-6.0.0/lib/python3.12/site-packages:/nix/store/qy2r1hr08yh0bj4k00iq8hya3fwb0s5m-python3.12-msgspec-0.19.0/lib/python3.12/site-packages:/nix/store/cfb23jvm74rbm1dl8lvgawbqpham3551-python3.12-gguf-0.17.1/lib/python3.12/site-packages:/nix/store/ay0znairvbbnx6alra5mw5szywj9xp9k-python3.12-poetry-core-2.1.3/lib/python3.12/site-packages:/nix/store/vaxyyqby74l710qdiar07r3d9g556qyv-python3.12-einops-0.8.1/lib/python3.12/site-packages:/nix/store/l6jz7r3mkvzcjq0by5fzfwp9jlrvfnzn-python3.12-importlib-metadata-8.7.0/lib/python3.12/site-packages:/nix/store/cvkzcf32ivybhjxw19837lv273l1kpxa-python3.12-toml-0.10.2/lib/python3.12/site-packages:/nix/store/4pjzhz9m4hs31ldwypmzw9iyzsxbd6pp-python3.12-zipp-3.23.0/lib/python3.12/site-packages:/nix/store/cz9z6nwk0wngwlnraid39dsa3hc187g4-python3.12-partial-json-parser-0.2.1.1.post6/lib/python3.12/site-packages:/nix/store/j053hbshq1wg2ifs5jdzdxcjxkghn3s5-python3.12-compressed-tensors-0.11.0/lib/python3.12/site-packages:/nix/store/1x68j82z0gsvqhaa31grj25g00hpmgfp-python3.12-frozendict-2.4.6/lib/python3.12/site-packages:/nix/store/2iqry5q7zhm1qmsq88shr8vxz4f6hc66-python3.12-mistral-common-1.8.4/lib/python3.12/site-packages:/nix/store/sy5v5jrd1kmwbb55s1gpkk24sx0r3xdb-python3.12-pydantic-extra-types-2.10.5/lib/python3.12/site-packages:/nix/store/kvjsw9rzj3sy3mmclnhcx5y7z3hskbyp-python3.12-torchaudio-2.8.0/lib/python3.12/site-packages:/nix/store/jzfii52dw49bkbxhckwrq6g9j9nz31p5-python3.12-torchvision-0.23.0/lib/python3.12/site-packages:/nix/store/82dwmr13kwlixbi41a16wijgsnn1ryr4-python3.12-scipy-1.16.1/lib/python3.12/site-packages:/nix/store/k14bksi5q4jiapx0anypy3b5f77k3g19-python3.12-uvicorn-0.35.0/lib/python3.12/site-packages:/nix/store/k08bvlay7cr851vby37d1ba5ig3zm95p-python3.12-xformers-0.0.30/lib/python3.12/site-packages:/nix/store/knqdwq17765jpz4a655y3fnhj2f0dah5-python3.12-xgrammar-0.1.24/lib/python3.12/site-packages:/nix/store/dn60r908c3j05m8c7v5ipl83icyvranb-python3.12-numba-0.62.0/lib/python3.12/site-packages:/nix/store/b8i493nyp3l8j5zdxvhvp63c3zv8adzz-python3.12-llvmlite-0.45.0/lib/python3.12/site-packages:/nix/store/jgq0cgny22cc1z263pbfhfwqsh45332l-python3.12-opentelemetry-sdk-1.34.0/lib/python3.12/site-packages:/nix/store/k4mffq3bxxqhhx6m923cszwxxl574gap-python3.12-opentelemetry-api-1.34.0/lib/python3.12/site-packages:/nix/store/v0ajrxaqw9k8wp0x314v80ni2ljaj15m-python3.12-deprecated-1.2.18/lib/python3.12/site-packages:/nix/store/4ycqfkys9167ffngcn2lf6avqhh3dfdw-python3.12-wrapt-1.17.2/lib/python3.12/site-packages:/nix/store/8i7a2y7wx2iglz4dyhh9r3gkhz1wpdgp-python3.12-opentelemetry-semantic-conventions-0.55b0/lib/python3.12/site-packages:/nix/store/xzfl8l4k0p3c0dcn2lcvw7jh3ykm6fya-python3.12-opentelemetry-exporter-otlp-1.34.0/lib/python3.12/site-packages:/nix/store/rk9q6qsngyzxy35l8pblx7kzpkxk7qbb-python3.12-opentelemetry-exporter-otlp-proto-grpc-1.34.0/lib/python3.12/site-packages:/nix/store/f9pspwrbzhqw3199xqc2y221q13pd49n-python3.12-googleapis-common-protos-3.31.3/lib/python3.12/site-packages:/nix/store/acdwxvn6jwxxs321nbi2fp4i0bipk35p-python3.12-opentelemetry-exporter-otlp-proto-common-1.34.0/lib/python3.12/site-packages:/nix/store/2k4lpwl744ab5ac9s9knw3aiw61v0ilz-python3.12-opentelemetry-proto-1.34.0/lib/python3.12/site-packages:/nix/store/zwfzcqppjj36bqggqg0s90hr0lkqizv5-python3.12-opentelemetry-exporter-otlp-proto-http-1.34.0/lib/python3.12/site-packages:/nix/store/79fb1y0r7jlm5fn86rh3qp2bmq0qcl3h-python3.12-bitsandbytes-0.47.0/lib/python3.12/site-packages:/nix/store/k67cxls64vczby694wpna4k3ixy7lq93-python3.12-setproctitle-1.3.6/lib/python3.12/site-packages:/nix/store/zc5ndla3fl25wnm1500b8ax58vpfwjsq-python3.12-openai-harmony-0.0.4/lib/python3.12/site-packages:/nix/store/krvkfnhm8fccw3g48xqvhqqk8ghb17kg-python3.12-httptools-0.6.4/lib/python3.12/site-packages:/nix/store/g32a1bn0m67k9g5yrph2mnv9pjfng8ml-python3.12-python-dotenv-1.1.1/lib/python3.12/site-packages:/nix/store/5ym5v5drby3waf7rg1475azlh352p2i6-python3.12-uvloop-0.21.0/lib/python3.12/site-packages:/nix/store/m6d1bkzknigm3abn7mpp9hpjd3drskh8-python3.12-py-libnuma-1.2/lib/python3.12/site-packages:/nix/store/0f6snlzlrqhg3pfrlb1kd7xwf8swm8yn-python3.12-cupy-13.6.0/lib/python3.12/site-packages:/nix/store/9ql478q8650fgrkdn3c71q8q2grj6jh8-python3.12-fastrlock-0.8.3/lib/python3.12/site-packages:/nix/store/ncihg2p25p5pza6fsiqxsfcspvfzd7zz-python3.12-pynvml-13.0.1/lib/python3.12/site-packages:/nix/store/8dlgbc7pqmviarigvjdrzd4mdd1ijnfb-python3.12-nvidia-ml-py-13.580.82/lib/python3.12/site-packages:/nix/store/i9ml5dkz33s3pkgywm7hzl69xsp5q4fc-python3.12-flashinfer-0.3.1/lib/python3.12/site-packages:/nix/store/iacgwr7m9gq0si9pjfsycx2fr3zcnsn1-python3.12-tabulate-0.9.0/lib/python3.12/site-packages:/nix/store/zsbkvanzzx4dd5va9ivsx83rs12d4dsv-python3-3.12.11/lib/python312.zip:/nix/store/zsbkvanzzx4dd5va9ivsx83rs12d4dsv-python3-3.12.11/lib/python3.12:/nix/store/zsbkvanzzx4dd5va9ivsx83rs12d4dsv-python3-3.12.11/lib/python3.12/lib-dynload:/nix/store/59pcizvg9ihq2kblyvpq1b4qxk0v7c48-python3.12-setuptools-80.9.0/lib/python3.12/site-packages/setuptools/_vendor&#x27;, &#x27;-DFETCHCONTENT_BASE_DIR=/build/source/.deps&#x27;, &#x27;-DNVCC_THREADS=1&#x27;, &#x27;-DCMAKE_JOB_POOL_COMPILE:STRING=compile&#x27;, &#x27;-DCMAKE_JOB_POOLS:STRING=compile=32&#x27;, &#x27;-DCMAKE_CUDA_COMPILER=/nix/store/l3zic8g7k0mzs3m3d0hw9l2awswfw61r-cuda_nvcc-12.8.93/bin/nvcc&#x27;]&#x27; returned non-zero exit status 1.

ERROR Backend subprocess exited when trying to invoke build_wheel</pre>
</details>
<details>
<summary>python313Packages.vllm</summary>
<pre>in configure
  File &quot;/nix/store/829wb290i87wngxlh404klwxql5v18p4-python3-3.13.7/lib/python3.13/subprocess.py&quot;, line 419, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command &#x27;[&#x27;cmake&#x27;, &#x27;/build/source&#x27;, &#x27;-G&#x27;, &#x27;Ninja&#x27;, &#x27;-DCMAKE_BUILD_TYPE=RelWithDebInfo&#x27;, &#x27;-DVLLM_TARGET_DEVICE=cuda&#x27;, &#x27;-DFETCHCONTENT_SOURCE_DIR_CUTLASS:STRING=/nix/store/n3bkhnw519zjw9g66l37l1yks6cqri0y-source&#x27;, &#x27;-DFLASH_MLA_SRC_DIR:STRING=/nix/store/ab1x9ra2sc9k9r2nxsrhhcm0izncxbsz-flashmla-1.0.0&#x27;, &#x27;-DVLLM_FLASH_ATTN_SRC_DIR:STRING=/nix/store/s9xqh0qzpk19hx5n0i52r2rf2fpxhw9a-vllm-flash-attn-2.7.4.post1&#x27;, &#x27;-DTORCH_CUDA_ARCH_LIST:STRING=7.5;8.0;8.6;8.9;9.0;10.0;12.0&#x27;, &#x27;-DCUTLASS_NVCC_ARCHS_ENABLED:STRING=75;80;86;89;90;100;120&#x27;, &#x27;-DCUDA_TOOLKIT_ROOT_DIR:STRING=/nix/store/vq0njavy3k2lkxv06yq2g5jq93a44r09-cuda-merged-12.8&#x27;, &#x27;-DCAFFE2_USE_CUDNN:STRING=ON&#x27;, &#x27;-DCAFFE2_USE_CUFILE:STRING=ON&#x27;, &#x27;-DCUTLASS_ENABLE_CUBLAS:STRING=ON&#x27;, &#x27;-DCUDA_HOST_COMPILER=/nix/store/fp97arzydz32c5gln0fia194n5mynwkw-gcc-wrapper-14.3.0/bin/c++&#x27;, &#x27;-DCMAKE_CUDA_HOST_COMPILER=/nix/store/fp97arzydz32c5gln0fia194n5mynwkw-gcc-wrapper-14.3.0/bin/c++&#x27;, &#x27;-DVLLM_PYTHON_EXECUTABLE=/nix/store/829wb290i87wngxlh404klwxql5v18p4-python3-3.13.7/bin/python3.13&#x27;, &#x27;-DVLLM_PYTHON_PATH=/nix/store/829wb290i87wngxlh404klwxql5v18p4-python3-3.13.7/lib/python3.13/site-packages:/nix/store/w54x75brfndymjy9v4845nsfagbnbjny-python3.13-wheel-0.46.1/lib/python3.13/site-packages:/nix/store/wjncbjj9g21a04ms4p3k2p7dhdik86yy-python3.13-packaging-25.0/lib/python3.13/site-packages:/nix/store/2z5phbjrv3bwi2r97h11xa9b2hqfywbv-python3.13-installer-0.7.0/lib/python3.13/site-packages:/nix/store/n3i9hx4y2y9hmv9rdrlibaqh7075drv4-python3.13-cmake-3.31.7/lib/python3.13/site-packages:/nix/store/gypqas7cdfsswnjw2n7bpajx00vkw4h6-python3.13-jinja2-3.1.6/lib/python3.13/site-packages:/nix/store/wrapwh5pjhacvjddhjln8hrpv3pzdwc1-python3.13-markupsafe-3.0.2/lib/python3.13/site-packages:/nix/store/bn4dg72qnk37cihrsdgg7x20m2nrx8ck-python3.13-ninja-1.13.1/lib/python3.13/site-packages:/nix/store/ybb7fxf4gx0y3yqynh39sr6qhkzp832i-python3.13-setuptools-80.9.0/lib/python3.13/site-packages:/nix/store/blfcj6g6lbihdmk97p8mvkn987jxxb62-python3.13-setuptools-scm-9.0.1/lib/python3.13/site-packages:/nix/store/ma47j6ifsckcbyz6x4ibnv6k3ld82q9q-python3.13-typing-extensions-4.14.1/lib/python3.13/site-packages:/nix/store/65cicsyy464f40vjf6cyyirf9vbk7gvl-python3.13-astunparse-1.6.3/lib/python3.13/site-packages:/nix/store/apdq89gzkdyr6j8xa8nvphks5n0hqpnj-python3.13-six-1.17.0/lib/python3.13/site-packages:/nix/store/q1r2wc2qq7q4fly8ks7w8k4af9g2z1nn-python3.13-expecttest-0.2.1/lib/python3.13/site-packages:/nix/store/15qv7ll4x5921lj2iassxiyd3vh9l3dw-python3.13-filelock-3.18.0/lib/python3.13/site-packages:/nix/store/nzj1mjh0qdkcdimzvi3lpyld1f2ybcv5-python3.13-fsspec-2025.3.2/lib/python3.13/site-packages:/nix/store/b6c78k0jpa27xq81m45vl8gxbmj7mh1b-python3.13-hypothesis-6.136.9/lib/python3.13/site-packages:/nix/store/8f4c2g6xil1a3ma083n0igajg9dahdwp-python3.13-attrs-25.3.0/lib/python3.13/site-packages:/nix/store/0x7p1clcafhawj4vvamcd9g2v3xlbw7p-python3.13-sortedcontainers-2.4.0/lib/python3.13/site-packages:/nix/store/v2pnnbb7vz17canng63k2k40l1vs0mp5-python3.13-networkx-3.5/lib/python3.13/site-packages:/nix/store/15558mx0vsjvgkk5p26a52y685ipbbpy-python3.13-psutil-7.0.0/lib/python3.13/site-packages:/nix/store/0bb2jwvapkd7k8f8blg5a8yfb7f5qsgd-python3.13-pyyaml-6.0.2/lib/python3.13/site-packages:/nix/store/myqvdcn01i0ckf9ghx9z7jlvwkbkjzxs-python3.13-requests-2.32.4/lib/python3.13/site-packages:/nix/store/rs70njhg79czsvp6jgm31g3xp0rl6w3f-python3.13-certifi-2025.07.14/lib/python3.13/site-packages:/nix/store/l71is046x0rj88sp37h8rj2ci2vb2rjg-python3.13-charset-normalizer-3.4.3/lib/python3.13/site-packages:/nix/store/klfh83ikviwpnpffp3rp2dav8vq0nw2n-python3.13-idna-3.10/lib/python3.13/site-packages:/nix/store/dmh9kcaliifdkk6nah6fnwfkm93vvykb-python3.13-urllib3-2.5.0/lib/python3.13/site-packages:/nix/store/yf3xcp727g12mkqmy8w9m08mg230h259-python3.13-sympy-1.14.0/lib/python3.13/site-packages:/nix/store/glyvwz2i263nsd7rmzl17ds53cvkjv5r-python3.13-mpmath-1.3.0/lib/python3.13/site-packages:/nix/store/rlf5k06yhqimnr9z09ynp5m7l5knr4cr-python3.13-types-dataclasses-0.6.6/lib/python3.13/site-packages:/nix/store/v0ks7f6nlzlngr3wdf2xr7r8b8yb0agc-python3.13-pillow-11.3.0/lib/python3.13/site-packages:/nix/store/1a0gbz18bjfk097pj6pmm70rw849p3gr-python3.13-tensorboard-2.20.0/lib/python3.13/site-packages:/nix/store/y6cw7wa1jgr2ri4mv47yxbg1imjscdn6-python3.13-absl-py-2.3.1/lib/python3.13/site-packages:/nix/store/2a6a4dwqgbgix87j2l3v8cc61yv9gbyp-python3.13-protobuf-6.32.0/lib/python3.13/site-packages:/nix/store/y7c9qyz7zzbbk3ghmywc02ifyqy5c9rc-python3.13-grpcio-1.74.0/lib/python3.13/site-packages:/nix/store/1d6yi4j6mc4n6f0pbr83nqjwfbr9rhah-python3.13-markdown-3.8.2/lib/python3.13/site-packages:/nix/store/xq23bm8sim4q9jbmb1dbfmwg9hrjwpkj-python3.13-numpy-2.3.2/lib/python3.13/site-packages:/nix/store/b369hgkms00l79xrf0dqx91a3q0kjkpm-python3.13-tensorboard-data-server-0.7.2/lib/python3.13/site-packages:/nix/store/k12xzz87yisw8hck5v3dzjqcfcwh1s5r-python3.13-werkzeug-3.1.3/lib/python3.13/site-packages:/nix/store/vndgg8kn0lk6mg1j7r3b1a0888fimar5-python3.13-standard-imghdr-3.13.0/lib/python3.13/site-packages:/nix/store/06krd0xnaa10d7c4iir4kqs70k2a1k0d-python3.13-pybind11-2.13.6/lib/python3.13/site-packages:/nix/store/vn4riw518m0i4205vbl52sxbz68c7b9w-python3.13-triton-3.4.0/lib/python3.13/site-packages:/nix/store/68gndzhm5gbvrv8cswfqic8n9srpfw4q-python3.13-torch-2.8.0/lib/python3.13/site-packages:/nix/store/b5s7n9xlb9vk5c761liyvhzawy76i35a-python3.13-cffi-1.17.1/lib/python3.13/site-packages:/nix/store/v94k1nmmidrmsb5c98rnwhkj242pbxin-python3.13-pycparser-2.22/lib/python3.13/site-packages:/nix/store/mghw13mqi0hihx2189fiydzj50gkx729-python3.13-aioprometheus-unstable-2023-03-14/lib/python3.13/site-packages:/nix/store/4aix8k3sx484w2bnx5yrgf44fzwh3zc7-python3.13-orjson-3.10.18/lib/python3.13/site-packages:/nix/store/2r9rqchcsyykfq54s5zvgy1iyzh3859m-python3.13-quantile-python-1.1/lib/python3.13/site-packages:/nix/store/8m60ka2n87h8ijlmsy1gfrjs64ppkr41-python3.13-blake3-1.0.7/lib/python3.13/site-packages:/nix/store/2wk6als3xmvs5giqln1r3s3x3l8kp189-python3.13-cachetools-6.1.0/lib/python3.13/site-packages:/nix/store/gblqbwr5lh5gpz32r8mbnsqxm8hz9r3p-python3.13-cbor2-5.6.5/lib/python3.13/site-packages:/nix/store/7xzchl4j0jjq45qjqjlrp3c64qjzbxc8-python3.13-depyf-0.19.0/lib/python3.13/site-packages:/nix/store/pi6ghmw4h5916a65wpvs6n37gr46v4d4-python3.13-astor-0.8.1/lib/python3.13/site-packages:/nix/store/zn8s03dldkdfz99vksgc2n3pq5pdf8i3-python3.13-dill-0.4.0/lib/python3.13/site-packages:/nix/store/znf677jcjwsmhccn1b5kdn5jdpfmhv5c-python3.13-fastapi-0.116.1/lib/python3.13/site-packages:/nix/store/svji8gzvgriax1c4lbawbn17irmz66lg-python3.13-starlette-0.47.2/lib/python3.13/site-packages:/nix/store/isf04w7cfxy6lvj4l0kq7k1igj6a398i-python3.13-anyio-4.10.0/lib/python3.13/site-packages:/nix/store/krsra4vjiwqcdxlkahhisaij49fdr233-python3.13-sniffio-1.3.1/lib/python3.13/site-packages:/nix/store/1bz0mpj234bg0fgrm64nqyds6w3x7i72-python3.13-pydantic-2.11.7/lib/python3.13/site-packages:/nix/store/rr74lm96ay1pb6ms5kjh2cxq80pz19rb-python3.13-annotated-types-0.7.0/lib/python3.13/site-packages:/nix/store/4w6ln0321ls00a9abmadj4zqgmjl5v0c-python3.13-pydantic-core-2.33.2/lib/python3.13/site-packages:/nix/store/wwkgiw9hm0vbbxzxm008zlh5zd67hgdz-python3.13-typing-inspection-0.4.1/lib/python3.13/site-packages:/nix/store/qz5cxs12557ihr43r27gisikhy54p2qw-python3.13-llguidance-1.2.0/lib/python3.13/site-packages:/nix/store/zg9gz1w78wirw45c4dimvlszgnam0sfs-python3.13-lm-format-enforcer-0.11.3/lib/python3.13/site-packages:/nix/store/73kg4vj8mlj6x5gagfdnlg9v2n8zhh6y-python3.13-interegular-0.3.3/lib/python3.13/site-packages:/nix/store/vgf5aq66x11n30l8m969asjnjw2ajmqh-python3.13-openai-1.101.0/lib/python3.13/site-packages:/nix/store/02x0fn0zgcwp6aggwg4wrh770ri2shi5-python3.13-distro-1.9.0/lib/python3.13/site-packages:/nix/store/jj7i2kx1d4gkb8rh224ib8b701c8qvg4-python3.13-httpx-0.28.1/lib/python3.13/site-packages:/nix/store/7w34kfnkcbsikgskjh5dmlk6d3718721-python3.13-httpcore-1.0.9/lib/python3.13/site-packages:/nix/store/0ps92fybd6zv285zffhbg77ds0a70pyb-python3.13-h11-0.16.0/lib/python3.13/site-packages:/nix/store/bkhvxzrizsk6dl57w1qxj79dz4820pkw-python3.13-jiter-0.8.2/lib/python3.13/site-packages:/nix/store/6adbhbqx86arizm6rcxzk2ds50ymdr7a-python3.13-tqdm-4.67.1/lib/python3.13/site-packages:/nix/store/g1nf3naxydxfhh72qijaci2aima0pr88-python3.13-aiohttp-3.12.15/lib/python3.13/site-packages:/nix/store/62bif12f0ijbinwnk50nwadaccw4qrir-python3.13-aiohappyeyeballs-2.6.1/lib/python3.13/site-packages:/nix/store/7jvp9mzm3rgs63bckaf6ns4sbqavsjlb-python3.13-aiosignal-1.4.0/lib/python3.13/site-packages:/nix/store/4xiyd0yd88xh9a3z8pc8ksfig9aa3gqn-python3.13-frozenlist-1.7.0/lib/python3.13/site-packages:/nix/store/fyfz33hn72f9jax4nf052lr260bkp0vh-python3.13-async-timeout-5.0.1/lib/python3.13/site-packages:/nix/store/9gkkc8ck2h557ydwnb0cq1gwkl42cqhg-python3.13-multidict-6.6.4/lib/python3.13/site-packages:/nix/store/q4s97gw8w5ic40wpzid0ia1g70fa0jc5-python3.13-propcache-0.3.2/lib/python3.13/site-packages:/nix/store/yjg9a830maawxfwwkkb2al5wf75wbpfa-python3.13-yarl-1.21.1/lib/python3.13/site-packages:/nix/store/h1pv76j5f3g0cz4gaxd1rs39018ksryx-python3.13-aiodns-3.5.0/lib/python3.13/site-packages:/nix/store/vzlwp7jj4hx9r5si83plf19hiqjbqsnl-python3.13-pycares-4.9.0/lib/python3.13/site-packages:/nix/store/15axb1pby6904lcyx80840lcmqd91rs6-python3.13-brotli-1.1.0/lib/python3.13/site-packages:/nix/store/20cw7wm5jv6mmcypbk8ahmb3r8zmsp8z-python3.13-httpx-aiohttp-0.1.8/lib/python3.13/site-packages:/nix/store/kdg351mrw16ynh7xxjykzp8r5vsz6zp9-python3.13-websockets-15.0.1/lib/python3.13/site-packages:/nix/store/lc6g9dpg85789nwfa9yk0wd9jar63piv-python3.13-sounddevice-0.5.2/lib/python3.13/site-packages:/nix/store/jd09r9mvmd77s8qq6p5dpzjnfdhb3g14-python3.13-opencv-python-headless-4.12.0/lib/python3.13/site-packages:/nix/store/z4jpwlg5c5s76s3gkvpdhn1jsyha27m1-opencv-4.12.0/lib/python3.13/site-packages:/nix/store/8paz6m7w09wzfa6655p9r4p3142d3d7j-python3.13-outlines-1.2.3/lib/python3.13/site-packages:/nix/store/b648bq5fm8cm2qib5jla5cq2ybyxx52q-python3.13-airportsdata-20250909.5/lib/python3.13/site-packages:/nix/store/dyrx4bmk2laajs1rjb21d9mb5vrqph75-python3.13-cloudpickle-3.1.1/lib/python3.13/site-packages:/nix/store/bagxrjjip5pj7jndkmb9rp529rmnsyic-python3.13-datasets-4.0.0/lib/python3.13/site-packages:/nix/store/7phr0778xpgmzv9hj7ma6xalx8i6y5n6-python3.13-huggingface-hub-0.35.0/lib/python3.13/site-packages:/nix/store/pl1wziahlaa6d4ssj6s92rdf2r3b8cix-python3.13-hf-xet-1.1.10/lib/python3.13/site-packages:/nix/store/j6pb5za3dk1pis9wmn3waqcnqr5ykvgz-python3.13-multiprocess-0.70.18/lib/python3.13/site-packages:/nix/store/l8hwi2c1p70hii57dpgydzzdzanpc81m-python3.13-pandas-2.3.1/lib/python3.13/site-packages:/nix/store/iyrn2r7lhw0f029vvg9zv8ansans72a4-python3.13-python-dateutil-2.9.0.post0/lib/python3.13/site-packages:/nix/store/7g3rb19vdan1nzf1vsf08nphq809amfx-python3.13-pytz-2025.2/lib/python3.13/site-packages:/nix/store/xrgc2hzgvvx9aw5m24zh2py4q0h2q1nx-python3.13-tzdata-2025.2/lib/python3.13/site-packages:/nix/store/nvlznr8flwkxj3d8q992bv9cgyhxpfxa-python3.13-pyarrow-20.0.0/lib/python3.13/site-packages:/nix/store/ba8jgpax23vsnkgk9gv4247kp243p3bm-python3.13-responses-0.25.7/lib/python3.13/site-packages:/nix/store/kp42rrwhm45jighkd37h090hg2ammhg4-python3.13-types-pyyaml-6.0.12.20250516/lib/python3.13/site-packages:/nix/store/b9saaqc8bwq66pq850y8gc52jby9whlh-python3.13-types-toml-0.10.8.20240310/lib/python3.13/site-packages:/nix/store/9agiwiqbwcsardsnkdzk3wb2qy7xc6w8-python3.13-xxhash-3.5.0/lib/python3.13/site-packages:/nix/store/b5ng2aga3ij9sdi9wlb3ypgy3gnx2mf6-python3.13-diskcache-5.6.3/lib/python3.13/site-packages:/nix/store/njc733ciw3rlxzkh4ki9yf8wr6hyammj-python3.13-genson-1.3.0/lib/python3.13/site-packages:/nix/store/xl3cvwp2hil4hr6z6kq2d5rbv67dqa2b-python3.13-iso3166-2.1.1/lib/python3.13/site-packages:/nix/store/xzn1qcg2xhzsyf4xwsa3prgppfqb8fn6-python3.13-jsonschema-4.25.0/lib/python3.13/site-packages:/nix/store/hcgmp3pqyxjzm6bnf942qvqif1g0z4qh-python3.13-jsonpath-ng-1.7.0/lib/python3.13/site-packages:/nix/store/zc9wcs3df3g8r8vqqfb3hpf5050a88bm-python3.13-ply-3.11/lib/python3.13/site-packages:/nix/store/2bfdrzrwsrv9ylbm2vnm3i83xwqp8ykh-python3.13-jsonschema-specifications-2025.4.1/lib/python3.13/site-packages:/nix/store/5j6y26qdxfgqr8bwhfnbgza2j9l4q4db-python3.13-referencing-0.36.2/lib/python3.13/site-packages:/nix/store/xr47wlgdmzq1ina942vgpx13zl5rzv3k-python3.13-rpds-py-0.25.0/lib/python3.13/site-packages:/nix/store/6m4j32imzam7gfn81g95yybqxgavrf4l-python3.13-lark-1.2.2/lib/python3.13/site-packages:/nix/store/6yhzm8pscw1104jxhrlgb7ir28718fgv-python3.13-regex-2025.7.34/lib/python3.13/site-packages:/nix/store/wmldlaxgijsm4ahzcp2x5a2fpc7khzm7-python3.13-nest-asyncio-1.6.0/lib/python3.13/site-packages:/nix/store/v8p1irlc5crcxkb5h82bdpn4gc12wdgm-python3.13-outlines-core-0.2.11/lib/python3.13/site-packages:/nix/store/66z666vr6a51y1cni6c30kkws8jr7f27-python3.13-pycountry-24.6.1/lib/python3.13/site-packages:/nix/store/50c05rnbw5v20j2lcfp19d222g0093a0-python3.13-transformers-4.56.2/lib/python3.13/site-packages:/nix/store/dy34nmddnyb6zjn6abwrcypxai0hrdan-python3.13-tokenizers-0.22.1/lib/python3.13/site-packages:/nix/store/32h2i9q23fz1mjk7nw34c9y0smrccl3g-python3.13-safetensors-0.6.2/lib/python3.13/site-packages:/nix/store/94hx6wsarqbbr6ig9lxjh442vsysz4bm-python3.13-prometheus-fastapi-instrumentator-7.1.0/lib/python3.13/site-packages:/nix/store/4js8hvxk6gwlsl572v81rcmhb15di5h2-python3.13-prometheus-client-0.22.1/lib/python3.13/site-packages:/nix/store/nkmzslgl2bwlpf5mrlv9x4p36dxipc6r-python3.13-py-cpuinfo-9.0.0/lib/python3.13/site-packages:/nix/store/vlh69zp1z3vxpiz6h4yv75hxi8xlya7k-python3.13-pybase64-1.4.2/lib/python3.13/site-packages:/nix/store/p1sbdbxlbb5qa3pfkx63xjam5chn6hng-python3.13-python-json-logger-3.3.0/lib/python3.13/site-packages:/nix/store/n0wbd5c533vdnqd0hn3cf990fmm8fd8k-python3.13-python-multipart-0.0.20/lib/python3.13/site-packages:/nix/store/wz1sbkf7wjl383wrc9m5v51w6yxilg7i-python3.13-pyzmq-27.0.1/lib/python3.13/site-packages:/nix/store/k00z5ik929sxdjmraszs0r1wwwdgx02w-python3.13-ray-2.49.2/lib/python3.13/site-packages:/nix/store/blgjyz871q0dibbawn9c0dmgn812d62s-python3.13-click-8.1.8/lib/python3.13/site-packages:/nix/store/kilfjhx0h5hifcpfmc2wf4qy8zc86v56-python3.13-msgpack-1.1.1/lib/python3.13/site-packages:/nix/store/5i2yzdv6xbxny4qwnad505j4dv2i63gn-python3.13-watchfiles-1.0.5/lib/python3.13/site-packages:/nix/store/3lgbkj2ls24gwajcqvdsx145sfdw4si8-python3.13-sentencepiece-0.2.1/lib/python3.13/site-packages:/nix/store/z21z9rf312z5jlq7xp95q2g2p71rm600-python3.13-tiktoken-0.9.0/lib/python3.13/site-packages:/nix/store/d5s6rhg6cwsriq6gbig5vnhh4ynslc0b-python3.13-blobfile-3.0.0/lib/python3.13/site-packages:/nix/store/kvq4l70gc2flslaal8mwnqlhicx4gvmd-python3.13-pycryptodomex-3.23.0/lib/python3.13/site-packages:/nix/store/j7j5y8pary2zjkiybzv2k98m7n1c5dfq-python3.13-lxml-6.0.0/lib/python3.13/site-packages:/nix/store/x1nnkphp4rsw0mmgks7qs9gpxazajijj-python3.13-msgspec-0.19.0/lib/python3.13/site-packages:/nix/store/5c14rqhisqbghsxy09qmkmfxmxd42529-python3.13-gguf-0.17.1/lib/python3.13/site-packages:/nix/store/n3k1xbmr7yspfq79kz2mid77k53820xa-python3.13-poetry-core-2.1.3/lib/python3.13/site-packages:/nix/store/r9mx2pxmq7ljay20rdzm4nmlqmyrmr1q-python3.13-einops-0.8.1/lib/python3.13/site-packages:/nix/store/p7yxvg351q2x6w1lya6ib7vlr6xcgf23-python3.13-importlib-metadata-8.7.0/lib/python3.13/site-packages:/nix/store/zqi630my03zvg24lxdbjw6clqgxc5c7i-python3.13-toml-0.10.2/lib/python3.13/site-packages:/nix/store/d5b7fvh73l4smj86ra611z7s1z0785y1-python3.13-zipp-3.23.0/lib/python3.13/site-packages:/nix/store/4h2hmi0mi9q1gg96kc2vxcywpb06aiw8-python3.13-partial-json-parser-0.2.1.1.post6/lib/python3.13/site-packages:/nix/store/im6j4fiaxh1s3lyi6iggx4n2cv5lygph-python3.13-compressed-tensors-0.11.0/lib/python3.13/site-packages:/nix/store/8xwgsy761x4bsv32n7cxiz3yx588rd8w-python3.13-frozendict-2.4.6/lib/python3.13/site-packages:/nix/store/gbaz3a46kdslbda27i2krjb3wrdi9hps-python3.13-mistral-common-1.8.4/lib/python3.13/site-packages:/nix/store/3zz7wrcn9nk5cvwmycz4kcj5ccwk55bz-python3.13-pydantic-extra-types-2.10.5/lib/python3.13/site-packages:/nix/store/8y06v3hqpsdz1db84b2v0vjgfhfr511a-python3.13-torchaudio-2.8.0/lib/python3.13/site-packages:/nix/store/8w64zljzh9ia2r6ymj8wb6jmijapifca-python3.13-torchvision-0.23.0/lib/python3.13/site-packages:/nix/store/bmmba1rr0hs8my3hsfmrbxr3g8ivvbws-python3.13-scipy-1.16.1/lib/python3.13/site-packages:/nix/store/w4rq6p638drsjk6fq0637dfmxxcaaqy7-python3.13-uvicorn-0.35.0/lib/python3.13/site-packages:/nix/store/q9mc6pkaghlnfqip76dfawphski8677r-python3.13-xformers-0.0.30/lib/python3.13/site-packages:/nix/store/x4a8n57pnvpsqnr76cmk7dz180vcg982-python3.13-xgrammar-0.1.24/lib/python3.13/site-packages:/nix/store/lmbb96jd7c06b07vzbzprviyzqdljdd8-python3.13-numba-0.62.0/lib/python3.13/site-packages:/nix/store/nkivk5lc579zfgkbnr6f3mqhxw9ps1pz-python3.13-llvmlite-0.45.0/lib/python3.13/site-packages:/nix/store/ynmy885xzqh010b1p6g4x2i2pjfm3cnr-python3.13-opentelemetry-sdk-1.34.0/lib/python3.13/site-packages:/nix/store/qmh3h378ial3a8pgmsnh53jwphmszya9-python3.13-opentelemetry-api-1.34.0/lib/python3.13/site-packages:/nix/store/l1g0yamssy01r9466is21f1lcyvr83zn-python3.13-deprecated-1.2.18/lib/python3.13/site-packages:/nix/store/x8zrsas74vxiwp97n54c5k9mar3amkr0-python3.13-wrapt-1.17.2/lib/python3.13/site-packages:/nix/store/qxyzar4gxv4v3kmi64n9511wwqay053p-python3.13-opentelemetry-semantic-conventions-0.55b0/lib/python3.13/site-packages:/nix/store/kil4ff7mhl1k0dn1i7vhbpjgn39d2qvk-python3.13-opentelemetry-exporter-otlp-1.34.0/lib/python3.13/site-packages:/nix/store/cnxkmjb421h9npg78zy2pv59143fyqfb-python3.13-opentelemetry-exporter-otlp-proto-grpc-1.34.0/lib/python3.13/site-packages:/nix/store/yav1qwvr86bii8wgxgdn83228hi3385n-python3.13-googleapis-common-protos-3.31.3/lib/python3.13/site-packages:/nix/store/ihiqpm2dlddcj5sdxgxp9d8qh6g0fwfv-python3.13-opentelemetry-exporter-otlp-proto-common-1.34.0/lib/python3.13/site-packages:/nix/store/9spbmhk93fs76gf3yd91k2mp5qbnv5in-python3.13-opentelemetry-proto-1.34.0/lib/python3.13/site-packages:/nix/store/4zlskskbiiv93hhf336ziskjwrz6nb0q-python3.13-opentelemetry-exporter-otlp-proto-http-1.34.0/lib/python3.13/site-packages:/nix/store/5qpx67inmsi4gqg02vrrrrl0in9x525l-python3.13-bitsandbytes-0.47.0/lib/python3.13/site-packages:/nix/store/ql79l5wp1cpd40qablvixlnf3qvr8vvc-python3.13-setproctitle-1.3.6/lib/python3.13/site-packages:/nix/store/swy70x6bdkqhq6bzyn5mh8rc2bihdh7y-python3.13-openai-harmony-0.0.4/lib/python3.13/site-packages:/nix/store/l1xslzzybwcspzr42pimyypyh3kkf1rh-python3.13-httptools-0.6.4/lib/python3.13/site-packages:/nix/store/yrpvm1nfi9yjsfzrxfr2fpd0yr96l4aa-python3.13-python-dotenv-1.1.1/lib/python3.13/site-packages:/nix/store/dczn6vg418bvw3p4365awbki1qpqpq75-python3.13-uvloop-0.21.0/lib/python3.13/site-packages:/nix/store/h21bcirncqxa2rsfz0y5dqid66336p1z-python3.13-py-libnuma-1.2/lib/python3.13/site-packages:/nix/store/afylbzcdcwlq7nrdliy5pbh02g755mkl-python3.13-cupy-13.6.0/lib/python3.13/site-packages:/nix/store/lkwz58w4w2rb8xz8l756vagvcjarmy9x-python3.13-fastrlock-0.8.3/lib/python3.13/site-packages:/nix/store/f3jqw7j44c9xs1lkzxfaqsppx1486zcx-python3.13-pynvml-13.0.1/lib/python3.13/site-packages:/nix/store/dz2ym17nqdap4ffmxal5qfcdzizr31ib-python3.13-nvidia-ml-py-13.580.82/lib/python3.13/site-packages:/nix/store/b6y8zx2f4jkb3rswxykiqdssc7y6rp8g-python3.13-flashinfer-0.3.1/lib/python3.13/site-packages:/nix/store/nphari8f9w0a04ykd12npkq9zzimgqhb-python3.13-tabulate-0.9.0/lib/python3.13/site-packages:/nix/store/829wb290i87wngxlh404klwxql5v18p4-python3-3.13.7/lib/python313.zip:/nix/store/829wb290i87wngxlh404klwxql5v18p4-python3-3.13.7/lib/python3.13:/nix/store/829wb290i87wngxlh404klwxql5v18p4-python3-3.13.7/lib/python3.13/lib-dynload:/nix/store/ybb7fxf4gx0y3yqynh39sr6qhkzp832i-python3.13-setuptools-80.9.0/lib/python3.13/site-packages/setuptools/_vendor&#x27;, &#x27;-DFETCHCONTENT_BASE_DIR=/build/source/.deps&#x27;, &#x27;-DNVCC_THREADS=1&#x27;, &#x27;-DCMAKE_JOB_POOL_COMPILE:STRING=compile&#x27;, &#x27;-DCMAKE_JOB_POOLS:STRING=compile=32&#x27;, &#x27;-DCMAKE_CUDA_COMPILER=/nix/store/l3zic8g7k0mzs3m3d0hw9l2awswfw61r-cuda_nvcc-12.8.93/bin/nvcc&#x27;]&#x27; returned non-zero exit status 1.

ERROR Backend subprocess exited when trying to invoke build_wheel</pre>
</details>
</details>

GaetanLepage commented:
Thanks for the PR @daniel-fahey !
Indeed, `vllm` fails to build with cuda support. Have you checked if the flashmla&co dependencies need to be updated?

GaetanLepage commented:
Now the CPU build fails with:
```
/build/source/csrc/cpu/dnnl_helper.cpp:4:10: fatal error: common/memory_desc.hpp: No such file or directory
    4 | #include "common/memory_desc.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
```

GaetanLepage commented:
How was it building on this branch earlier today???

daniel-fahey commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 447722`
Commit: `e7fb8960241ab2211c471004bcd2b04683767c33`

---
### `x86_64-linux`
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>python312Packages.kserve</li>
    <li>python312Packages.kserve.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>vllm (python312Packages.vllm)</li>
    <li>vllm.dist (python312Packages.vllm.dist)</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 6 packages built:</summary>
  <ul>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vllm.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>vllm</summary>
<pre>[nixbuild.net] Cached build failure. The id of the previously failed build is 4565380. See this link for the failed build details and logs: https://nixbuild.net/builds/4565380?t=EtkBCm8KBWJ1aWxkCgpidWlsZDpyZWFkGAMiCQoHCAcSAxD5BSINCgsIBBIHOgUKAxiBCDImCiQKAggbEgYIBRICCAUaFgoECgIIBQoICgYg9eyp0wYKBBoCCAAyFgoUCgIIGxIOCAISAxiACBIFEITTlgISJAgAEiAGRu1AM51kzPYGl4nShP2nj4SwLZdIQEZhBaIO6KS15xpATQzlvJ7Y-SLutaWbRFkbSrm3qOrarrHr60v_1kS5vx08FGnOtkZSZjwNvMDFTJwiO1O-MfmWac0BNAXtrNz8ASIiCiAuWrCOa6uQpEanWP7s1LJ8iKWZhtugtJoeboULO7JD1w==
[nixbuild.net] To turn off build failure caching, see https://docs.nixbuild.net/settings/#reuse-build-failures</pre>
</details>
</details>

daniel-fahey commented:
> How was it building on this branch earlier today???

This builds the working Python 3.12 CPU version:
```console
[daniel@laptop:~/Source/nixpkgs]$ nix-build -I nixpkgs=https://github.com/daniel-fahey/nixpkgs/archive/fbc1629b8775a9eeb932f1b79d14fa973adb67ec.tar.gz   --expr 'with import <nixpkgs> { }; python312Packages.vllm'
/nix/store/zdm5svz7ygij3dj13djsizm1zggbnapw-python3.12-vllm-0.10.2
```

And this is the broken one:
```console
[daniel@laptop:~/Source/nixpkgs]$ nix-build -I nixpkgs=https://github.com/daniel-fahey/nixpkgs/archive/e7fb8960241ab2211c471004bcd2b04683767c33.tar.gz   --expr 'with import <nixpkgs> { }; python312Packages.vllm'
this derivation will be built:
  /nix/store/j98dgdx87kjahfj73cd0wvzp3c0px4dh-python3.12-vllm-0.10.2.drv
building '/nix/store/j98dgdx87kjahfj73cd0wvzp3c0px4dh-python3.12-vllm-0.10.2.drv' on 'ssh://eu.nixbuild.net'...
copying 0 paths...
[nixbuild.net] Cached build failure. The id of the previously failed build is 4565380. See this link for the failed build details and logs: https://nixbuild.net/builds/4565380?t=EtkBCm8KBWJ1aWxkCgpidWlsZDpyZWFkGAMiCQoHCAcSAxD5BSINCgsIBBIHOgUKAxiBCDImCiQKAggbEgYIBRICCAUaFgoECgIIBQoICgYg9eyp0wYKBBoCCAAyFgoUCgIIGxIOCAISAxiACBIFEITTlgISJAgAEiAGRu1AM51kzPYGl4nShP2nj4SwLZdIQEZhBaIO6KS15xpATQzlvJ7Y-SLutaWbRFkbSrm3qOrarrHr60v_1kS5vx08FGnOtkZSZjwNvMDFTJwiO1O-MfmWac0BNAXtrNz8ASIiCiAuWrCOa6uQpEanWP7s1LJ8iKWZhtugtJoeboULO7JD1w==
[nixbuild.net] To turn off build failure caching, see https://docs.nixbuild.net/settings/#reuse-build-failures
error: build of '/nix/store/j98dgdx87kjahfj73cd0wvzp3c0px4dh-python3.12-vllm-0.10.2.drv' on 'ssh://eu.nixbuild.net' failed: Cached build failure (build 4565380): Cannot build '/nix/store/j98dgdx87kjahfj73cd0wvzp3c0px4dh-python3.12-vllm-0.10.2.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/cfj51613p4smfza7150mzsk21smfrzxl-python3.12-vllm-0.10.2
         /nix/store/l2gjwc39xss9hg63rbpgh66is87ay4iw-python3.12-vllm-0.10.2-dist
error: Cannot build '/nix/store/j98dgdx87kjahfj73cd0wvzp3c0px4dh-python3.12-vllm-0.10.2.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/cfj51613p4smfza7150mzsk21smfrzxl-python3.12-vllm-0.10.2
         /nix/store/l2gjwc39xss9hg63rbpgh66is87ay4iw-python3.12-vllm-0.10.2-dist
       Last 2 log lines:
       > [nixbuild.net] Cached build failure. The id of the previously failed build is 4565380. See this link for the failed build details and logs: https://nixbuild.net/builds/4565380?t=EtkBCm8KBWJ1aWxkCgpidWlsZDpyZWFkGAMiCQoHCAcSAxD5BSINCgsIBBIHOgUKAxiBCDImCiQKAggbEgYIBRICCAUaFgoECgIIBQoICgYg9eyp0wYKBBoCCAAyFgoUCgIIGxIOCAISAxiACBIFEITTlgISJAgAEiAGRu1AM51kzPYGl4nShP2nj4SwLZdIQEZhBaIO6KS15xpATQzlvJ7Y-SLutaWbRFkbSrm3qOrarrHr60v_1kS5vx08FGnOtkZSZjwNvMDFTJwiO1O-MfmWac0BNAXtrNz8ASIiCiAuWrCOa6uQpEanWP7s1LJ8iKWZhtugtJoeboULO7JD1w==
       > [nixbuild.net] To turn off build failure caching, see https://docs.nixbuild.net/settings/#reuse-build-failures
       For full logs, run:
         nix log /nix/store/j98dgdx87kjahfj73cd0wvzp3c0px4dh-python3.12-vllm-0.10.2.drv
```

Comparing the build inputs:
```console
[daniel@laptop:~/Source/nixpkgs]$ diff \
> <(nix derivation show --impure -I nixpkgs=https://github.com/daniel-fahey/nixpkgs/archive/fbc1629b8775a9eeb932f1b79d14fa973adb67ec.tar.gz --expr 'with import <nixpkgs> { }; python312Packages.vllm' | jq -r '.[] | .inputDrvs | keys[]') \
> <(nix derivation show --impure -I nixpkgs=https://github.com/daniel-fahey/nixpkgs/archive/e7fb8960241ab2211c471004bcd2b04683767c33.tar.gz --expr 'with import <nixpkgs> { }; python312Packages.vllm' | jq -r '.[] | .inputDrvs | keys[]')
57a58
> /nix/store/lnybvwfry02293c564dm33kv7ash7bck-python3.12-blake3-1.0.7.drv
71d71
< /nix/store/rc213bx4f77s732gj49y51r87z5mcjzy-python3.12-blake3-1.0.6.drv
```
Blake3 was updated in 29c1e9e8b18198c2d7a4cb6db92758f7b08e66b8, maybe somehow relevant? It _is_ a dependency.

daniel-fahey commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 447722`
Commit: `5d9245450c60b4c2eb4582b4024e89776e9960fa`

---
### `x86_64-linux`
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>python312Packages.kserve</li>
    <li>python312Packages.kserve.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>vllm (python312Packages.vllm)</li>
    <li>vllm.dist (python312Packages.vllm.dist)</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 6 packages built:</summary>
  <ul>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vllm.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>vllm</summary>
<pre>[nixbuild.net] Cached build failure. The id of the previously failed build is 4565380. See this link for the failed build details and logs: https://nixbuild.net/builds/4565380?t=EtkBCm8KBWJ1aWxkCgpidWlsZDpyZWFkGAMiCQoHCAcSAxD5BSINCgsIBBIHOgUKAxiBCDImCiQKAggbEgYIBRICCAUaFgoECgIIBQoICgYg9pGq0wYKBBoCCAAyFgoUCgIIGxIOCAISAxiACBIFEITTlgISJAgAEiC0VHyZUI0P3GLNjwL3hFjQnoHZxiphXPrPB3nlxnkJIRpA3DUGBgrNTalTX909Npvy8M00k21nq-8mTHbjr6HLEfVMJeFpCxGAbVnFwNS53GZOt7IxDyvkXmeEbPkJ9fvNByIiCiAo2Ic3aY4hmmdqs_VqHbWYl_I4QyywZCymznDzI99pIw==
[nixbuild.net] To turn off build failure caching, see https://docs.nixbuild.net/settings/#reuse-build-failures</pre>
</details>
</details>

Commit:
python3Packages.vllm: add daniel-fahey to maintainers

Commit:
python3Packages.vllm: 0.10.1.1 -> 0.10.2

Notably adds Python 3.13 support (see vllm-project/vllm#13164) among many other things (https://github.com/vllm-project/vllm/releases/tag/v0.10.2).

Co-authored-by: Gaétan Lepage <gaetan@glepage.com>

GaetanLepage commented:
> Please file an issue with vllm, perhaps its authors have more insight.

Could you do so @daniel-fahey please? You may then add a link to the issue in the derivation.

GaetanLepage reviewed (approved):
Thanks @daniel-fahey !
